### ISSUE TITLE
Restyling of Department Browse

### ISSUE THREAD

**Issue by kzadrozny** [uchicago-library/library_website#20]
- Remove bulleted styling and make more visually interesting
- Suggestion of reorganizing by alpha by parent department
- Try combination of headers and zebra striping or accordion collapse

**Comment by kzadrozny** [comment]
EL requests:
- way to show what filters have been applied.

Will work with @johnjung to whiteboard filter/label options.

**Comment by elisabeth-long** [comment]
EL also requests:
- compactify the layout so you see more lines on the screen at once - right now one loses any sense of what division you are looking at because everything is so spread out

- Just browsing some interesting examples
Harvard puts the org in the sidebar as a facet engine http://library.harvard.edu/staff-directory#
Penn is pretty compact and starts with some TOCs: http://dla.library.upenn.edu/dla/staff/index.html
Duke also compact (though no showing depth: https://library.duke.edu/about/directory
Brown - I like their 'responsible for' line (but I know that is out of scope): http://library.brown.edu/about/deptlist.php

**Comment by kzadrozny** [comment]
In terms of layout, I think [Harvard](http://library.harvard.edu/staff-directory#) has some great ideas. The sidebar facets and the "see also" are great. I can see us using the "See also" position for our "quick numbers," which has been commented as being easy to miss in its current position.

Additionally, I like the way [NCSU's department page](https://www.lib.ncsu.edu/department/access-services) looks, specifically the top and right sidebar. I'm not a fan of the wall of faces and see us using the same row layout we have now.

**Comment by elisabeth-long** [comment]
Yes - I'd looked at NCSU, but had focused on the faces and not liked that.  Their, and Harvard's use of the full dept list in the side bar could make navigating around it very easy.

**Comment by johnjung** [comment]
Notes from David: the new mockup is a radical restyling- this should be one option of two. (Option "A".) It's useful for getting feedback on, but we should also make something that nudges the current display just a bit.

For option "B", make something that matches what we have used up until this point more closely. Bring the switcher back so people can switch back and forth between a heirarchical view and an alphabetical view of units. For the heirarchical view, show all levels of hierarchy as accordions- but if the user clicks the second level of hierarchy just expand everything. 

Instead of using facets for option "B", try using a sidebar that has links, like the sidebar links we use on the site right now. Here are some sidebar links he sketched out:
Staff and Departments
__Departments
____Hierarchical
____Alphabetical
__Staff
____Library
______Crerar
______...
____Subject
______Area and Cultural Studies
______...
(other links)

For all options: incorporate something so that users can get a department roster for a unit, but also click to go to the unit's URL, if they have one. Look for things that could be removed from the contact block in the department view. Use the staff view as the default. 

Regarding the division head information- this still looks like an entry in the list of units instead of information about the division. Is it possible to make the division head photo smaller? Could this information be included in the header for the division itself?

**Comment by kzadrozny** [comment]
Option 1: https://kzadrozny.github.io/LibPattern/directory-org.html
> Default would be all staff page. Sidebar includes old filters as links. Compressed department view, with toggle option for sub-departments.

Option 2: https://kzadrozny.github.io/LibPattern/directory-redux.html
> Staff & departments merged together with filters in sidebar.

**Comment by ecboettch** [comment]
from WCG, 8/30:

Departmental view will be option 1, with the following changes:

- Flat organization
- Alphabetical
- No sidebar - filtering will be done at the top
- Link to org chart 

In addition, the search on these pages should search both departments and staff, and the default view should be staff.

[full WCG discussion](https://loop.lib.uchicago.edu/groups/web-content-group/meeting-minutes/2017/83017/) on Loop.

**Comment by johnjung** [comment]
Kathy, can you include a public link to that mock? :)

**Comment by johnjung** [comment]
Thanks, Emma- that looks like it, based on the web content minutes. Based on the minutes I'll consider this approved. I'll code it up soon.
